### ISSUE TITLE
[ .net 5 ] : EnsureCreated always use ODBC

### ISSUE THREAD

**Issue by nicovil** [bubibubi/EntityFrameworkCore.Jet#122]
When executing ~~`Migrate` or `MigrateAsync`~~ `EnsureCreated` and `EnsureCreatedAsync`, it doesn't matter which connection string we use, the provider will always use ODBC.

In my case, I'm working with OLEDB and I want `EnsureCreated` to use OLEDB too, the reason is that I'm using some datatypes that doesn't works well with ODBC but everything works perfectly with OLEDB.

Not 100% sure, but looks like the `JetDatabaseCreator` invoke the `CreateEmptyConnection` method which for some reason retrieves ODBC by default.

As extra information, if you use `Migrate` and `MigrateAsync`, it uses OLEDB and migration just creates the database correctly.

**Comment by nicovil** [comment]
For more information, I'm using this workaround to create the database when using JET provider:

```
var databaseCreator = JetDatabaseCreator.CreateInstance(SchemaProviderType.Adox);
databaseCreator.CreateDatabase(theConnectionString);
```

This one works and creates the empty database, so later I can apply the Migration with **Context.Database.Migrate()** without any problem.

**Comment by lauxjpn** [comment]
@nicovil Sorry can you clarify a bit, since the following two statements seem contradict each other:

> When executing Migrate or MigrateAsync, it doesn't matter which connection string we use, the provider will always use ODBC.
> [...]
> As extra information, if you use Migrate and MigrateAsync, it uses OLEDB and migration just creates the database correctly.

Thanks!

**Comment by lauxjpn** [comment]
@nicovil Thanks, got it now! I'll take a look at this.

**Comment by ChrisJollyAU** [comment]
Fixed as part of #135
